### PR TITLE
[8.0][FIX] According with the decorator change on super method

### PR DESCRIPTION
**Pull request by adrienpeiffer**
According with the decorator change on super method (@api.model -> @api.multi) in #196

### CODE REVIEW

File: account_payment_blocking/model/payment_order_create.py [hunk 1]
```diff
@@ -27,8 +27,8 @@ class payment_order_create(orm.TransientModel):
     _inherit = 'payment.order.create'
 
     def extend_payment_order_domain(
-            self, cr, uid, payment_order, domain, context=None):
+            self, cr, uid, ids, payment_order, domain, context=None):
```
Comment (pedrobaeza) [inline account_payment_blocking/model/payment_order_create.py:30]: Isn't better to migrate this to new API?

File: account_payment_mode_term/models/payment_order_create.py [hunk 2]
```diff
@@ -31,9 +31,9 @@ class payment_order_create(orm.TransientModel):
     _inherit = 'payment.order.create'
 
     def extend_payment_order_domain(
-            self, cr, uid, payment_order, domain, context=None):
+            self, cr, uid, ids, payment_order, domain, context=None):
         super(payment_order_create, self).extend_payment_order_domain(
-            cr, uid, payment_order, domain, context=context)
+            cr, uid, ids, payment_order, domain, context=context)
```
Comment (pedrobaeza) [inline account_payment_mode_term/models/payment_order_create.py:36]: The same

File: account_payment_blocking/model/payment_order_create.py (hunk 1 above)
Comment (adrienpeiffer) [inline account_payment_blocking/model/payment_order_create.py:30]: Hi @pedrobaeza,
I think it's a bit out of scope. It's just a fix.

Comment (pedrobaeza) [inline account_payment_blocking/model/payment_order_create.py:30]: As this change is only 2 lines, I see it very simple:

```
@api.multi
def extend_payment_order_domain(
        self, payment_order, domain):
    super(payment_order_create, self).extend_payment_order_domain(
        payment_order, domain)
...
```

Comment (adrienpeiffer) [inline account_payment_blocking/model/payment_order_create.py:30]: Yes but there something else in this module

Comment (pedrobaeza) [inline account_payment_blocking/model/payment_order_create.py:30]: You can migrate only these files, not the entire module. Old and new API can coexist, even in the same module or file

**Comment by sbidoul** [comment]
New API migration is never trivial. We always have surprises. And we have no resources to do additional new API convertion in this module right now.

:+1:

**Comment by lmignon** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Not for this case. I ask because it is very simple (I already paste the code), but no problem in letting as is.

:+1: Merging.

**Pull request merged**